pkg/controller/controller_utils.go

```diff
@@ -585,10 +585,10 @@ func (r RealPodControl) createPods(nodeName, namespace string, template *v1.PodT
 	}
 	accessor, err := meta.Accessor(object)
 	if err != nil {
-		klog.Errorf("parentObject does not have ObjectMeta, %v", err)
+		klog.ErrorS(err, "parentObject does not have ObjectMeta")
```

Comment:
```suggestion
		klog.ErrorS(err, "ParentObject does not have ObjectMeta")
```

Reply:
Please do not capitalize error messages. We try to adhere to https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
Fixed.